pkg/admin/brokerclient.go

```diff
@@ -690,3 +696,69 @@ func configEntriesToAPIConfigs(
 
 	return apiConfigs
 }
+
+// GetACLs gets full information about each ACL in the cluster.
+func (c *BrokerAdminClient) GetACLs(
+	ctx context.Context,
+	filter kafka.ACLFilter,
+) ([]ACLInfo, error) {
+	req := kafka.DescribeACLsRequest{
+		Filter: filter,
+	}
+	log.Debugf("DescribeACLs request: %+v", req)
+
+	resp, err := c.client.DescribeACLs(ctx, &req)
+	log.Debugf("DescribeACLs response: %+v (%+v)", resp, err)
+	if err != nil {
+		return nil, err
+	}
+
+	if resp.Error != nil {
+		return nil, resp.Error
+	}
+
+	aclinfos := []ACLInfo{}
+
+	for _, resource := range resp.Resources {
+		for _, acl := range resource.ACLs {
+			aclinfos = append(aclinfos, ACLInfo{
+				ResourceType:   ResourceType(resource.ResourceType),
+				ResourceName:   resource.ResourceName,
+				PatternType:    PatternType(resource.PatternType),
+				Principal:      acl.Principal,
+				Host:           acl.Host,
+				Operation:      ACLOperationType(acl.Operation),
+				PermissionType: ACLPermissionType(acl.PermissionType),
+			})
+		}
+	}
+
+	return aclinfos, nil
+}
+
+// CreateACL creates an ACL in the cluster.
+func (c *BrokerAdminClient) CreateACL(
```

Comment:
This can't be used by the user yet but this will be required in future work and adding it now made testing much better